doc_src/prompt.rst

```diff
@@ -148,10 +148,50 @@ And it looks like:
 .. parsed-literal::
     :class: highlight
 
+    :green:`~/M/L/Oneknowing`>false
     :green:`~/M/L/Oneknowing`\ :red:`[1]`>_
 
 after we run ``false`` (which returns 1).
 
+Transient prompt
+----------------
+
+Fish supports transient prompts, which let you to customize the prompt shown in the scrollback. This is handy for decluttering, as you can show less information in the final prompt than the transient (active) one.
+
+To enable transient prompt functionality, set the ``fish_transient_prompt`` variable to 1::
+
+  set -g fish_transient_prompt 1
+
+With this set, fish re-runs ``fish_prompt`` (and ``fish_right_prompt``) with the ``--final-rendering`` argument before pushing commandline to scrollback. We can use this to display only the current directory name in the final prompt::
```

Comment:
prompt.rst is a tutorial document, try to keep this language simple.

"With this set, fish re-runs ``fish_prompt`` and ``fish_right_prompt`` with a ``--final-rendering`` argument before running the next commandline.

So you can use it to declutter your old prompts. For example if you want to see only the current directory name when you scroll up: